Header header
uint8 turn_signal_state
bool engine_kill
bool honk
bool parking_brake
uint64 bytes_written
float64 timestamp

